kernel/linux/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=kernel-packages
+PKG_FLAGS:=hold
+
+PKG_MAINTAINER:=Alexandru Ardelean <ardeleanalex@gmail.com>
+
+PKG_BUILD_DIR:=$(KERNEL_BUILD_DIR)/packages
+SCAN_DEPS=modules/*.mk $(TOPDIR)/target/linux/*/modules.mk $(TOPDIR)/include/netfilter.mk
```

Comment:
Where does `SCAN_DEPS` get used?

Reply:
tbh, I don't know
after grep-ing through OpenWrt's source code there are a few places where this shows  up ;

Reply:
i thought i had a more complete reply about this;
so `SCAN_DEPS` seems to be part of some [currently] black-magic build environment variables for dependency manipulation;
[ black-magic because i did not try to understand it yet ]